addons/binding/org.openhab.binding.modbus/README.md

```diff
@@ -152,16 +152,17 @@ With low baud rates and/or long read requests (that is, many items polled), ther
-| `length`   | integer | ✓        | (-)                | Number of registers, coils or discrete inputs to read.                                                                                                                                 |
-| `type`     | text    | ✓        | (-)                | Type of modbus items to poll. This matches directly to Modbus request type or function code (FC). Valid values are: `coil` (FC01), `discrete` (FC02), `holding`(FC03), `input` (FC04). |
-| `refresh`  | integer |          | `500`              | Poll interval in milliseconds. Use zero to disable automatic polling.                                                                                                                  |
-| `maxTries` | integer |          | `3`                | Maximum tries when reading. <br /><br />Number of tries when reading data, if some of the reading fail. For single try, enter 1.                                                       |
+| Parameter     | Type    | Required | Default if omitted | Description                                                                                                                                                                            |
+| ------------- | ------- | -------- | ------------------ | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `start`       | integer |          | `0`                | Address of the first register, coil, or discrete input to poll. Input as zero-based index number.                                                                                      |
+| `length`      | integer | ✓        | (-)                | Number of registers, coils or discrete inputs to read.                                                                                                                                 |
+| `type`        | text    | ✓        | (-)                | Type of modbus items to poll. This matches directly to Modbus request type or function code (FC). Valid values are: `coil` (FC01), `discrete` (FC02), `holding`(FC03), `input` (FC04). |
+| `refresh`     | integer |          | `500`              | Poll interval in milliseconds. Use zero to disable automatic polling.                                                                                                                  |
+| `maxTries`    | integer |          | `3`                | Maximum tries when reading. <br /><br />Number of tries when reading data, if some of the reading fail. For single try, enter 1.                                                       |
+| `cacheMillis` | integer |          | `10`               | Duration for data cache to be valid, in milliseconds. This cache is used only to serve `REFRESH`  commands. Use non-positive integer to disable the caching.                           |
```

Comment:
Default is now 50, right?

And disable is 0 according to your explanation on the config.

Reply:
fixed